sim/core/stats/stats.go

```diff
@@ -311,6 +311,7 @@ type PseudoStats struct {
 
 	// Important when unit is attacker or target
 	BlockValueMultiplier float64
+	BlockDamageReduction float64
```

Comment:
You should also remove the `BlockValue` Stat at the same time as you make this change, both in the enum at the top of this file as well as in the protos.